apiserver/test/e2e/service_server_e2e_test.go

```diff
@@ -369,23 +366,17 @@ func createTestServiceV2(t *testing.T, tCtx *End2EndTestingContext) *api.CreateR
 	require.NoError(t, err, "No error expected")
 	require.Nil(t, actualRPCStatus, "No RPC status expected")
 	require.NotNil(t, actualService, "A service is expected")
-	waitForRunningService(t, tCtx, actualService.Name)
+
+	// Check if the service was created successfully
+	checkRayServiceCreatedSuccessfully(t, tCtx, actualService.Name)
 
 	return testServiceRequest
 }
 
-func waitForRunningService(t *testing.T, tCtx *End2EndTestingContext, serviceName string) {
-	// wait for the service to be in a running state for 3 minutes
-	// if is not in that state, return an error
-	err := wait.PollUntilContextTimeout(tCtx.ctx, 500*time.Millisecond, 3*time.Minute, false, func(_ context.Context) (done bool, err error) {
-		rayService, err := tCtx.GetRayServiceByName(serviceName)
-		if err != nil {
-			return true, err
-		}
-		t.Logf("Found status of '%s' for ray service '%s'", rayService.Status.ServiceStatus, serviceName)
-		return rayService.Status.ServiceStatus == rayv1api.Running, nil
-	})
-	require.NoErrorf(t, err, "No error expected when getting ray service: '%s', err %v", serviceName, err)
+func checkRayServiceCreatedSuccessfully(t *testing.T, tCtx *End2EndTestingContext, serviceName string) {
+	rayService, err := tCtx.GetRayServiceByName(serviceName)
+	require.NoError(t, err, "No error expected")
```

Comment:
The error message doesn't provide additional information.
```suggestion
	require.NoError(t, err)
```